### DISCUSSION TITLE
python setup.py develop gives error

### DISCUSSION THREAD

**Discussion by nehagoyal1994** [Project-MONAI/MONAI#3012] [Q&A]
Hi,

I am trying to install monai in my conda env. 
Using Option2 to install from installation page; command **python setup.py develop** gives error:
![image](https://user-images.githubusercontent.com/71243457/134479624-1917bbf2-d744-49b5-87c5-ff9c3db19287.png)

I checked CUDA info:
![image](https://user-images.githubusercontent.com/71243457/134484758-e1f1a196-1112-4875-a0e2-61099ee5ad32.png)

Could you please guide how to install monai in the Conda env?

Thanks in advance

**Comment by wyli** [comment]
the error message is "unknown url type: https", it looks like your python environment is missing some components.

there are some instructions for conda from last year's bootcamp: https://github.com/Project-MONAI/MONAIBootcamp2020#set-up-environment-using-conda .
